TDD_Training/TDD_Training/FizzBuzzService.Tests/FizzBuzzServiceTests.cs

```diff
@@ -71,6 +71,54 @@ public void GivenNumberIsNotMultipleOfThreeOrFive_IsMultipleOf_ReturnsGivenNumbe
             //Assert
             Assert.Equal(expected, actual);
         }
+
+        [Theory]
+        [InlineData(26)]
+        [InlineData(104)]
+        [InlineData(143)]
+        public void GivenNumberIsMultipleOfThirteen_IsMultipleOf_ReturnsJazz(int value)
+        {
+            //Arrange
+            var expected = Convert.ToString(value);
+
+            //Act
+            var actual = _fizzBuzzService.CheckFizzBuzz(value);
+
+            //Assert
+            Assert.Equal(expected, actual);
+        }
+
+        [Theory]
+        [InlineData(46)]
+        [InlineData(161)]
+        [InlineData(253)]
+        public void GivenNumberIsMultipleOfTwentyThree_IsMultipleOf_ReturnsWiz(int value)
+        {
+            //Arrange
+            var expected = Convert.ToString(value);
+
+            //Act
+            var actual = _fizzBuzzService.CheckFizzBuzz(value);
+
+            //Assert
+            Assert.Equal(expected, actual);
+        }
+
+        [Theory]
+        [InlineData(0)]
+        [InlineData(-5)]
+        [InlineData(-157)]
+        public void GivenNumberIsNegativeOrZero_IsMultipleOf_ThrowsException(int value)
+        {
```

Comment:
Your test description can explain what type of exception is getting thrown

Reply:
Changed the test description